build-system/tasks/compile.js

```diff
@@ -386,6 +386,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
         only_closure_dependencies: true,
         output_wrapper: wrapper,
         create_source_map: intermediateFilename + '.map',
+        source_map_include_content: true,
```

Comment:
I assume this needs to be removed before merging?

Reply:
If this will cause an undesirable increase in the size of the `.map` file for normal minified builds, maybe put it behind a flag?